packages/@aws-cdk/aws-dynamodb-global/README.md

```diff
@@ -0,0 +1,30 @@
+## @aws-cdk/aws-dynamodb-global
+
+Here is a minimal deployable Global DynamoDB tables definition:
+
+```typescript
+import globaldynamodb = require('@aws-cdk/aws-dynamodb-global');
+const CONSTRUCT_NAME = 'aws-cdk-dynamodb-global';
+const TABLE_PARTITION_KEY: Attribute = { name: 'hashKey', type: AttributeType.String };
+
+const stack = new Stack();
+const globalTables = new GlobalTable(stack, CONSTRUCT_NAME, {
+    dynamoProps: {
```

Comment:
Would make more sense for the GlobalTableProps to inherit from TableProps so you don't need `dynamoProps` as a separate nested structure (we try to avoid nesting as much as possible)

Reply:
I ran into issues with trying to extend TableProps - I needed to do some fancy TypeScript-fu to change tableName from optional to required and jsii didn't accept the type change.  

The cleanest way I thought to make it work is to have GlobalDynamoDBProps just be a copy/paste of TableProps with just that change.  I'm open for specific suggestions on how to clean that up, though

Reply:
I implemented this with your suggested change from below